template/en/default/bug/process/results.html.tmpl

```diff
@@ -29,6 +29,7 @@
 [% PROCESS global/variables.none.tmpl %]
 
 [% UNLESS header_done %]
+  [% PROCESS bug_modal/header.html.tmpl %]
```

Comment:
Hey guys, I've spent a long time on this:
Unless I manually include the bug_modal header, it doesn't get included, causing everything to render wrong (`bug/process/header.html.tmpl` below should be including it given that `show_bug.cgi` choses to use the modal format). The old `process_bug.cgi` before this PR did things very similar to what I'm doing in show_bug.cgi and the headers were included just fine.

I will keep digging into this but wanted to give an update